KamerConnect.View.MAUI/Views/InterestsForm.xaml.cs

```diff
@@ -1,9 +1,86 @@
+using KamerConnect.Models;
+using KamerConnect.Services;
+using Npgsql;
+
 namespace KamerConnect.View.MAUI.Views;
 
 public partial class InterestsForm : ContentView
 {
-	public InterestsForm()
+	private readonly PersonService _personService;
+	private Person? _currentPerson;
+	private Personality? personality;
+	public Personality? Personality
+	{
+		get => personality;
+		set
+		{
+			personality = value;
+		}
+	}
+	
+	public InterestsForm(PersonService personService, Person person)
 	{
+		_personService = personService;
+		_currentPerson = person;
+		Personality = person.Personality;
+		CheckIfPersonalityFieldsAreNull(person);
+		BindingContext = Personality;
 		InitializeComponent();
 	}
+
+	private void Button_update_interests(object? sender, EventArgs e)
+	{
+		Guid personId = Guid.Parse(_currentPerson.Id);
+		_personService.InsertTableIfIdIfNotExist(personId, "personality");
+		ValidateIfAccountDetailsChanged(_currentPerson);
```

Comment:
Moeten we perse alleen de gewijzigde velden meesturen? De velden die niet gewijzigd zijn bevatten als het goed is gewoon de oude waarde dus kan je ook gewoon meesturen. Dan kan je gewoon het hele Person object meesturen naar de update functie.